pom.xml

```diff
@@ -1144,11 +1144,11 @@
             </goals>
           </execution>
         </executions>
-      </plugin>
+      </plugin> -->
       <plugin>
         <groupId>com.github.siom79.japicmp</groupId>
         <artifactId>japicmp-maven-plugin</artifactId>
-        <version>0.15.4</version>
+        <version>0.14.3</version>
```

Comment:
same

Reply:
The version `0.15.4` fails siren netty installation with the error:
`[ERROR] Failed to execute goal com.github.siom79.japicmp:japicmp-maven-plugin:0.15.4:cmp (default) on project netty-common: There is at least one incompatibility: io.netty.util.AsciiString:METHOD_ABSTRACT_ADDED_IN_IMPLEMENTED_INTERFACE,io.netty.util.collection.ByteObjectHashMap:METHOD_ABSTRACT_ADDED_IN_IMPLEMENTED_INTERFACE,io.netty.util.collection.CharObjectHashMap:METHOD_ABSTRACT_ADDED_IN_IMPLEMENTED_INTERFACE,io.netty.util.collection.IntObjectHashMap:METHOD_ABSTRACT_ADDED_IN_IMPLEMENTED_INTERFACE,io.netty.util.collection.LongObjectHashMap:METHOD_ABSTRACT_ADDED_IN_IMPLEMENTED_INTERFACE,io.netty.util.collection.ShortObjectHashMap:METHOD_ABSTRACT_ADDED_IN_IMPLEMENTED_INTERFACE,io.netty.util.internal.AppendableCharSequence:METHOD_ABSTRACT_ADDED_IN_IMPLEMENTED_INTERFACE -> [Help 1]
`

Reply:
I am not sure what this error means. I saw that same error reported in an issue of the maven plugin https://github.com/siom79/japicmp/issues/281
I tried locally with version `0.15.7` and I had no issue running the command `mvn install`. I think it's better to move it forward.

Reply:
Done https://github.com/sirensolutions/netty/pull/29/commits/80abb4e6388788f0599acd0787d7d542b179ae2c